### DISCUSSION TITLE
New Achievement Profile: Versatile

### DISCUSSION THREAD

**Discussion by kauaneiras** [community/community#74764] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

![github](https://github.com/kauaneiras/images/blob/146c28ade894486adb25304a9eb6af959cd162b3/pag1.png?raw=true)

Achievement Name: Versatile

Description: This achievement is awarded to programmers who demonstrate their versatility and adaptability by using a diverse range of programming languages and technologies.

I thought of representing the achievement with a chameleon because chameleons can change colors to match their surroundings. I thought this would be a fitting representation of a programmer who is skilled in multiple programming languages and technologies. The chameleon's ability to change colors would represent the programmer's ability to adapt to new challenges and environments.

Levels:
Bronze: Use 10 or more different programming languages or technologies.
Silver: Use 15 or more different programming languages or technologies.
Gold: Use 20 or more different programming languages or technologies.
Example of how the achievement would be displayed on a profile:

Achievement: Versatile
Level: Silver
Languages/Technologies: Python, Java, JavaScript, C++, R, SQL, HTML, CSS, C, Docker, Kubernetes, Assembly,Rust, Ruby, Cobol.
</br>
<div style="display: inline-flex;">
<img src="https://github.com/kauaneiras/images/blob/main/pag2.png?raw=true" width="200" height="200" alt="Imagem de um camaleão">
<img src="https://github.com/kauaneiras/images/blob/main/Pag3.png?raw=true" width="200" height="200" alt="Imagem de um camaleão">
<img src="https://github.com/kauaneiras/images/blob/main/pag4.png?raw=true" width="200" height="200" alt="Imagem de um camaleão">
</div>
</br>
I would be grateful if you could help me make this topic more visible to GitHub. You can suggest other ideas, images, or anything else that you think would be helpful. I am open to all suggestions.

**Comment by kauaneiras** [comment]
All suggestions, criticisms and ideas are welcome in this thread 🤗

**Reply by Jose0908** [reply]
The 'Versatile' achievement is a fantastic concept, and I think it should be considered for implementation by GitHub. It would be a great addition to recognize and encourage programmers who demonstrate versatility with various languages and technologies. Well done on the proposal!

**Comment by natyhtpsm** [comment]
@⁨Nathália⁩ 
The "Versatile" achievement not only showcases a programmer's ability to adapt to various challenges but also reflects their deep understanding of different programming paradigms and problem-solving approaches. It's a testament to their dedication to continuous learning and their commitment to staying at the forefront of the ever-evolving tech landscape. So, whether you're coding in Python, Java, C++, SQL, HTML, CSS, Docker, Kubernetes, Assembly, Ruby, or even Cobol, this achievement celebrates your relentless pursuit of programming excellence.

**Reply by kauaneiras** [reply]
🤩🤩🤩

**Comment by gabrielw6** [comment]
Good initiative! Broadness of programming knowledge should surely be rewarded, if not, at least recognized. Versatility implies a refinement of algorithmic thinking, which is what developers are paid to do, independent of the lexicon and syntactic sugar within each respective language. No language is ever the best in everything, and the versatile developer knows it.

**Comment by RodrigoTCLima** [comment]
Nice

**Comment by gabrielapivetta** [comment]
Great idea! I believe this award can showcase developers that truly understand the different programming paradigms. Usually when someone knows a good amount of languages, we can imply that they have a more solid foundation on good practices.
I would like to suggest to add the paradigm of each language, that way the achievement can also acknowledge those that adventured themselves in languages that are outside of their comfort zones.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @kauaneiras ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by kauaneiras** [reply]
Thank You!!
